Update container-service-connect.md

IanMoroney opened the pull request:
Replaced `az acs kubernetes` with `az aks` as the cli now has dedicated aks commands.

azurecla commented:
Hi __@changeworld__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@changeworld, thanks for the corrections!

IanMoroney committed:
Update container-service-connect.md

Replaced `az acs kubernetes` with `az aks` as the cli now has dedicated aks commands.
